code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -2,6 +2,7 @@
 	real_name = "Test Dummy"
 	status_flags = GODMODE|CANPUSH
 	has_huds = FALSE
+	no_vore = TRUE //VOREStation Add - Dummies don't need bellies.
```

Comment:
human_species_vr.dm?

Reply:
This one, yes, since that file already exists.